src/index.html

```diff
@@ -315,19 +315,19 @@ <h5>Member discount</h5>
   <div class="section-deals">
     <div class="container">
       <div class="row">
-          <div class="col-6 mleft">
+        <div class="col-12 col-sm-12 col-md-6 mleft">
               <img class="deals-left" src="./images/sofa.jpg">
               <div class="textphoto">
                 <p class="room">GUEST ROOM </p><p class="sofa">SOFA</p>
                 <p class="price">-20%</p>
               </div>
           </div>
-          <div class="col-6 mright">
+          <div class="col-12 col-sm-12 col-md-6 mright">
```

Comment:
Jeśli dobrze rozumiem / pamiętam (a mogłem zdążyć zapomnieć od naszej ostatniej rozmowy na ten temat) to chyba nie potrzebujesz "col-sm-12" bo punktem wyjściowym (od "zera) jest "col-12" i zmieniane jest dopiero przez "col-md-6".